### PR TITLE
S02 us7 game list page

### PR DESCRIPTION
**Pull request by JadeJolivet**

### CODE REVIEW

**Review by PaulBOURY — APPROVED** [review]

**Review by FGAT3 — CHANGES_REQUESTED** [review]
Let's use and import the images from the back ;)

I suggest creating global const like :

const BASE_URL = import.meta.env.VITE_API_URL;
const catalog = `${BASE_URL}/catalog`;

And changing your json's image src path to : `catalog/file-name.format`

File: client/src/pages/GameListPage/GameListPage.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import "../../scss/variables.scss";
```
Comment (FGAT3) [inline client/src/pages/GameListPage/GameListPage.scss:1]: You only need to import index.scss

File: client/src/pages/GameListPage/GameListPage.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+@import "../../scss/mixins.scss";
```
Comment (FGAT3) [inline client/src/pages/GameListPage/GameListPage.scss:2]: You only need to import index.scss

**Review by FGAT3 — APPROVED** [review]

File: client/src/components/GameListModal/GameListModal.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            </svg>
+          </label>
+        </Form>
+        <img
+          src={`${BASE_URL}/${game.image_demo}`}
+          alt={`Demo de ${game.title}`}
+          className="demo-image"
+        />
+        <h2 className="title-modal">{game.title}</h2>
+        <p>Date de sortie : {game.date_de_sortie}</p>
```
Comment (KevinDavoust) [inline client/src/components/GameListModal/GameListModal.jsx]: in english please 👩‍🏫

File: client/src/components/GameListModal/GameListModal.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            Voir la démo
+          </a>
+        )}
+      </article>
+    </section>
+  );
+}
+
+GameListModal.propTypes = {
+  open: PropTypes.bool.isRequired,
```
Comment (KevinDavoust) [inline client/src/components/GameListModal/GameListModal.jsx]: si c'est un booleen, plutôt isOpen à la place du verbe qui laisse imaginer une fonction 🤨

File: client/src/components/GameListModal/GameListModal.scss [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+.modal-open {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 999;
```
Comment (KevinDavoust) [inline client/src/components/GameListModal/GameListModal.scss]: mdr

**Review by KevinDavoust — CHANGES_REQUESTED** [review]

**Review by KevinDavoust — APPROVED** [review]

**Pull request merged**